packages/frontend-sfe/src/lib/environment.ts

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+export const env = {
+  url:
+    process.env.NEXT_PUBLIC_VERCEL_URL && process.env.NEXT_PUBLIC_VERCEL_ENV! === 'preview'
+      ? `https://${process.env.NEXT_PUBLIC_VERCEL_URL}`
+      : process.env.NEXT_PUBLIC_URL,
+  isProduction: process.env.NEXT_PUBLIC_PRODUCTION_MODE === 'true',
+
+  defaultChain: parseInt(process.env.NEXT_PUBLIC_DEFAULT_CHAIN!),
+  supportedChains: JSON.parse(process.env.NEXT_PUBLIC_SUPPORTED_CHAINS!),
+
+  rpcUrls: {
```

Comment:
Wouldn't it be cleaner to export a function
```ts
export function getRpcUrl(chainId: int): string?
```
that returns `` process.env[`NEXT_PUBLIC_RPC_{$chainId}`] `` if the key of `process.env` is defined and `undefined` if not?

Reply:
This function already exists in `wagmiClient.ts`. I like to keep my `environment.ts` as logic-free as possible. But that's just personal taste, we can think about this in the future.

Reply:
Ah I see, you define the raw data here and then create the actual getter in `wagmiClient` 👍🏻